### PR TITLE
REL-4043: PIT support for AEON part I

### PR DESCRIPTION
**Pull request by cquiroz**
The PIT needs to add support for multi facility proposals. This PR is the first part containing the model changes and the basic UI.
The UI needs refinements but I'm awaiting for clarification on the requirements

### CODE REVIEW

File: bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/view/partner/PartnerView.scala [hunk 1]
```diff
@@ -301,6 +322,94 @@ class PartnerView extends BorderPanel with BoundView[Proposal] {view =>
...
+              case MultiFacilityWrapper.Yes => Some(QueueProposalClass.multiFacility.set(q, Some(new MultiFacility())))
+              case MultiFacilityWrapper.No => Some(QueueProposalClass.multiFacility.set(q, None))
+              case _ => Some(q)
+            }
+            case Some(l: LargeProgramClass)           => model = selection.item match {
+              case MultiFacilityWrapper.Yes => Some(LargeProgramClass.multiFacility.set(l, Some(new MultiFacility())))
+              case MultiFacilityWrapper.No => Some(LargeProgramClass.multiFacility.set(l, None))
+              case _ => Some(l)
+            }
+            case _                                    => // shouldn't happen
```
Comment (swalker2m) [inline bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/view/partner/PartnerView.scala:385]: Maybe blow up with an exception?

File: bundle/edu.gemini.model.p1/src/main/xsd/ProposalClass.xsd [hunk 2]
```diff
@@ -52,6 +52,19 @@
         </xsd:complexContent>
     </xsd:complexType>
 
+    <xsd:complexType name="Facility">
+        <xsd:sequence>
+            <xsd:element name="name" type="xsd:string" minOccurs="0" maxOccurs="1"/>
+            <xsd:element name="aeonCompatible" type="xsd:boolean" minOccurs="0" maxOccurs="1"/>
+        </xsd:sequence>
+    </xsd:complexType>
```
Comment (swalker2m) [inline bundle/edu.gemini.model.p1/src/main/xsd/ProposalClass.xsd:60]: I'm confused by what feeds the `aeonCompatible` element.  I mean, how does that get set in a proposal document.

**Review by swalker2m — APPROVED** [review]
Looks good.  With fear and trepidation about the downstream impact, :+1:

File: bundle/edu.gemini.model.p1/src/main/xsd/ProposalClass.xsd (hunk 2 above)
Comment (cquiroz) [reply bundle/edu.gemini.model.p1/src/main/xsd/ProposalClass.xsd:60]: There is a UI (still pending) that will let you select from a number of facilities and potentially the time requested at each

**Comment by cquiroz** [comment]
This probably has relatively low impact downstream. The only thing that would change is the xml content and we wouldn't need to display this on the OT or take care when doing ITAC

**Comment by tpolecat** [comment]
> The only thing that would change is the xml content and we wouldn't need to display this on the OT or take care when doing ITAC

What's the point of doing it at all then? It's just cosmetic?

**Comment by cquiroz** [comment]
It will be used by TACs, I guess ITAC too but AFAICT is only relevant for humans checking the pdf summary rather than any actions done by code

**Pull request merged**